src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -95,6 +75,59 @@ def generate_config(self, daemon_spec: CephadmDaemonDeploySpec) -> Tuple[Dict[st
         }
         return config_file, sorted(deps)
 
+    def prepare_certificates(self, daemon_spec: CephadmDaemonDeploySpec) -> Tuple[str, str]:
+        cert_path = f'{daemon_spec.host}/grafana_crt'
+        key_path = f'{daemon_spec.host}/grafana_key'
+        cert = self.mgr.get_store(cert_path)
+        pkey = self.mgr.get_store(key_path)
+        is_empty_certificate = not (cert and pkey)
```

Comment:
NIT, but could we use something like `certs_present = (cert and pkey)` here and then just flip whether we add a `not` for the two current use cases of `is_empty_certificate` below? What we have here works but the fact that we have this bool that represents a negated scenario then we immediately check for the negation of it (`if not is_empty_certificate`) makes me feel like we could simplify a bit.

Reply:
@adk3798 changed the bool to a positive check